src/pretix/base/settings.py

```diff
@@ -321,7 +321,7 @@
         'default': LazyI18nString.from_gettext(ugettext_noop("""Hello,
 
 we did not yet receive a payment for your order for {event}.
-Please keep in mind that if we only guarantee your order if we receive
+Please keep in mind that we only guarantee your order if we receive
```

Comment:
Thanks, will fix on master was well

Reply:
No problem of course, it is a very nice and cool piece of software that you created! So thank you for that :smiley:. I can remove this grammar correction from this pull request or make it into a new one if you desire.

I will continue with the Dutch translation, also because we are planning to use it for our next event (we are using the software for events in our student music association ESMG Quadrivium). When at least all front-end texts have been translated I'll update this pull request.